src/services/api.js

```diff
@@ -1,130 +1,91 @@
 import axios from 'axios'
-import Cookies from 'js-cookie'
 import env from '../config/environment'
 
-export const getQueryValue = (queryString, key) => {
-  return getSplitValue(key, queryString.substring(1), '&', '=')
-}
-
 const getSplitValue = (key, raw, delim1, delim2) => {
   const vars = raw.split(delim1)
 
-  for (let i = 0; i < vars.length; i++) {
+  for (let i = 0; i < vars.length; i += 1) {
     const pair = vars[i].split(delim2)
-    if (pair.length !== 2) {
-      continue
-    }
-
-    const pairKey = pair[0].trim()
-    const pairValue = pair[1].trim()
-    if (pairKey === key && pairValue) {
-      return pairValue
+    if (pair.length === 2) {
+      const pairKey = pair[0].trim()
+      const pairValue = pair[1].trim()
+      if (pairKey === key && pairValue) {
+        return pairValue
+      }
     }
   }
 
   return null
 }
 
-export const deleteCookie = name => {
-  const domain = window.location.hostname
-  Cookies.remove(name, { domain })
-  if (Cookies.get(name)) {
-    console.warn(
-      `${name} cookie couldn't be removed - check that domain matches, etc.`
-    )
-  }
-}
+export const getQueryValue = (queryString, key) => getSplitValue(key, queryString.substring(1), '&', '=')
```

Comment:
this is fine, no reason to change it, but I'm surprised there's not a better solution for this than splitting and parsing on our own.

Reply:
the browser doesn't provide a native API for parsing queryString params so this kind of code is not uncommon. I'm not saying this is the best solution, but I was just making lint fixes and not changing the code itself.

Reply:
that is surprising!